contracts/lib/Organization.sol

```diff
@@ -16,22 +16,51 @@ pragma solidity ^0.5.0;
 
 import "./SafeMath.sol";
 import "./OrganizationInterface.sol";
-import "./IsMemberInterface.sol";
 
 
 /**
  * @title Organization contract.
  *
  * @notice The organization represents an entity that manages the
- *         mosaic ecosystem and therefore the Organization.sol contract holds
+ *         mosaic ecosystem and therefore the `Organization.sol` contract holds
  *         all the keys required to administer the mosaic contracts.
+ *         This contract supports the notion of an "admin" that can act on
+ *         behalf of the organization. When seen from the outside by consumers
+ *         of the `OrganizationInterface`, a notion of an admin does not exist.
  */
-contract Organization is OrganizationInterface, IsMemberInterface {
+contract Organization is OrganizationInterface {
+
+
+    /* Using */
+
     using SafeMath for uint256;
 
+
+    /* Events */
+
+    /** Emitted when a current owner initiates a change of ownership. */
+    event OwnershipTransferInitiated(address indexed proposedOwner);
+
+    /** Emitted when a new owner accepts the ownership transfer. */
+    event OwnershipTransferCompleted(address newOwner);
+
+    /** Emitted whenever an owner or admin changes the address of the admin. */
+    event AdminAddressChanged(address indexed newAdmin);
+
+    /** Emitted when a worker address was set. */
+    event WorkerSet(
+        address indexed worker,
+        uint256 expirationHeight,
+        uint256 remainingHeight
```

Comment:
Sorry for confusion. I would propose to *remove* remainingHeight parameter. Do not see a reason to have it.

Reply:
Worker can be set with higher expiration height. remainingHeight ensures event listener knows from the current block, for how many blocks worker is active for. 
I know It can be derived from expirationHeight, however we concluded remainingHeight to be present in the event .
If we think anyone of  expirationHeight or remainingHeight should be there, we can remove expirationHeight and keep remainingHeight because expirationHeight is part of setWorker method signature.

Reply:
By that argument we need neither. Expiration height is given and remaining height can be calculated. However, the events are not for the people who did the transactions, but for others to be notified about changes. In this case, I could imagine a worker wants to know that its key is now set as a valid worker. In that case, the worker would also be interested in the expiration height.
The remaining height (which we don't currently see as relevant) is simply `expirationHeight - blockNumber`, both of which are listed in the web3 log and are therefore easily accessible by anyone who reads the event.
Calculating and adding the "remaining height" parameter to the event costs gas that we can potentially save, unless someone can explain what it is for.

Reply:
Discussed this point with Martin on quick call. Also tried to find in the history why remainingHeight was added with Workers contract. As far as I remember it was requested by the KIT at that time with Workers/Pricer/Airdrop integration. Looking at the current situation(gas cost) we can keep expirationHeight and remove remainingHeight as it can be derived from expirationHeight. We can always add it if it's needed in future with a separate ticket. 👍 🚗

Reply:
✅ removed